components/internal-services/internal_service_service_account_token.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: internal-service-request-service-account-secret
```

Comment:
Somebody should be able to read the token - please create record in `components/authentication` (I know that you are almighty admin but still :stuck_out_tongue_winking_eye:)

Reply:
added